frontend/src/components/general/Footer.js

```diff
@@ -82,11 +88,17 @@ const SmallFooter = ({ className, noSpacingTop, noAbsolutePosition }) => {
   const classes = useStyles();
   const isNarrowScreen = useMediaQuery(theme => theme.breakpoints.down("xs"));
 
+  const trigger = useScrollTrigger({
+    disableHysteresis: false,
+    threshold: 100
+  });
+
   return (
     <Box
       component="footer"
       className={`${className} ${classes.root} ${!noSpacingTop &&
-        classes.spacingTop} ${!noAbsolutePosition && classes.absolutePosition}`}
+        classes.spacingTop} ${!noAbsolutePosition &&
+        (!trigger ? classes.relativePosition : classes.absolutePosition) }`}
```

Comment:
This should only be used on "Infinite Scroll" pages. Specifically on the `/browse` page.
A good solution would be to add a variable `showOnScrollUp` which can be passed by the browse page to `WideLayout` and then to the `Footer` component.